src/osemgrep/cli_login/Console_Spinner.mli

```diff
@@ -0,0 +1,5 @@
+val spinner : string array
```

Comment:
If this global does not need to be accessed outside this module, you can just remove it from the .mli, so
the API becomes simpler.